share/site/duckduckgo/spread.tx

```diff
@@ -1,57 +1 @@
-		<div class="spread-hero--text--wide" style="max-width:605px;">
-                <h1 class="hd-hr  spread-title--hero">#ComeToTheDuckSide</h1>
-		</div>
-	</div>
-</div>
-<div class="blk  blk--dark  blk--arr">
-	<div class="cw--c">
-		<h2 class="hd-hr  spread-title"><a href="https://duckduckgo.com/?t=spread" style="color:white;">DuckDuckGo</a> doesn't store your personal information.</h2>
-		
-		<div class="spread-info-wrap  fix">
-			<div class="third">
-				<div class="spread-info">
-					<noscript>
-						<img class="spread-img" title="You know you love us..." src="/assets/spread/love.png" />
-					</noscript>
-					<img class="spread-img  js-lazysvg  no-js__hide" title="You know you love us..." src="" data-src="/assets/spread/love" />
-					
-					<h4 class="spread-info__hd">We Love Our Privacy</h4>
-					<p class="spread-info__p">Stop being followed by creepy ads, being sold out to advertisers, and being charged more money based on your search history.</p>
-				</div>
-			</div>
-			<div class="third">
-				<div class="spread-info  spread-info--mid">
-					<noscript>
-						<img class="spread-img" title="so share us with your friends" src="/assets/spread/share.png" />
-					</noscript>
-					<img class="spread-img  js-lazysvg  no-js__hide" title="so share us with your friends" src="" data-src="/assets/spread/share" />
-					
-					<h4 class="spread-info__hd">Help Your Friends</h4>
-					<p class="spread-info__p">Let your friends and family know that private browsing mode is not the solution. Other search engines can and do still track your searches there.</p>
-				</div>
-			</div>
-			<div class="third">
-				<div class="spread-info">
-					<noscript>
-						<img class="spread-img" title="... and help us take off!" src="/assets/spread/growth.png" />
-					</noscript>
-					<img class="spread-img  js-lazysvg  no-js__hide" title="... and help us take off!" src="" data-src="/assets/spread/growth" />
-					
-					<h4 class="spread-info__hd"><: l('We Need Your Help') :></h4>
-					<p class="spread-info__p"><: l("When you get others to switch, we can do even more! We're up against giant corporations, yet we are making a difference with your help.") :></p>
-				</div>
-			</div>
-		</div>
-	</div>
-</div>
-<div class="blk  blk--arr  blk--alt">
-   <div class="spread-callout">
-                                   <h2 class="hd-hr  spread-callout__title">
-                                                     <: l("Please consider sharing DuckDuckGo:", '<br class="hide--screen-s">') :>
-                                                        </h2>
-        <a class="js-twitter-btn spread-callout__btn btn btn--primary--alt t-xl" href="https://twitter.com/?status=DuckDuckGo%20is%20my%20search%20engine%20of%20choice.%20%23ComeToTheDuckSide%20-%20we%20have%20privacy.%20https%3A%2F%2Fduckduckgo.com%2F%231">Tweet #ComeToTheDuckSide</a>
-
-        </div>
-</div>
+<div class="spread-page"></div>
```

Comment:
minor nit, can we add `js-spread-page` here and update accordingly when access this element via JS?